crest/Assets/Crest/Crest/Scripts/LodData/Shadows/LodDataMgrShadow.cs

```diff
@@ -120,6 +120,11 @@ internal override void OnDisable()
         {
             base.OnDisable();
 
+            {
```

Comment:
Not certain the additional scope is necessary but that's minor

Reply:
It's for adding a built-in renderer conditional downstream. Setting up the scope here reduces conflicts. I'll add a comment.